executing/executing.py

```diff
@@ -151,12 +170,17 @@ def get_instructions(co):
 
 class NotOneValueFound(Exception):
     def __init__(self,msg,values=[]):
+        # type: (str, Sequence) -> None
         self.values=values
         super(NotOneValueFound,self).__init__(msg)
 
+T = TypeVar('T')
+
 
 def only(it):
+    # type: (Union[Iterable[T], Sequence[T], Set[T]]) -> T
```

Comment:
Isn't `Iterable` generic enough?

Reply:
Yup, using that now. Didn't know `Iterable` well enough TBH!